README.md

```diff
@@ -143,24 +161,25 @@ of quantum machine learning.
 
 ----------------------------------------------------------------------------------------------------
 
-## Contribution Guidelines
+## How can I contribute?
 
 If you'd like to contribute to Qiskit, please take a look at our
 [contribution guidelines](https://github.com/qiskit-community/qiskit-machine-learning/blob/main/CONTRIBUTING.md).
-This project adheres to Qiskit's [code of conduct](https://github.com/qiskit-community/qiskit-machine-learning/blob/main/CODE_OF_CONDUCT.md).
-By participating, you are expected to uphold this code.
+This project adheres to the Qiskit [code of conduct](https://github.com/qiskit-community/qiskit-machine-learning/blob/main/CODE_OF_CONDUCT.md).
+By participating, you must uphold this code.
```

Comment:
The text we had here was modelled after what Qiskit has in its readme https://github.com/Qiskit/qiskit?tab=readme-ov-file#contribution-guidelines

> By participating, you are expected to uphold...

The code itself uses the same language 

>  all participants, including attendees, speakers, sponsors, volunteers, online contributors, and IBM employees are expected to...

as in we expect you to do this, if you do not then appropriate actions....

The main difference is around who, as in "we expect" versus "you must" where the former although it implies the latter of you its not done as a directive to you